hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/DiskBalancerService.java

```diff
@@ -333,7 +338,7 @@ public BackgroundTaskQueue getTasks() {
 
     for (int i = 0; i < availableTaskCount; i++) {
       Pair<HddsVolume, HddsVolume> pair = volumeChoosingPolicy
-          .chooseVolume(volumeSet, threshold, deltaSizes);
+          .chooseVolume(volumeSet, threshold);
```

Comment:
Keep the deltaSizes and use it in chooseVolume for volumes, otherwise deltaSizes is not used.